testutil/models.go

```diff
@@ -72,3 +59,37 @@ type Language struct {
 	Code string `gorm:"primarykey"`
 	Name string
 }
+
+type Coupon struct {
+	ID               int              `gorm:"primarykey; size:255"`
+	AppliesToProduct []*CouponProduct `gorm:"foreignKey:CouponId;constraint:OnDelete:CASCADE"`
+	AmountOff        uint32           `gorm:"column:amount_off"`
```

Comment:
I noticed in a different PR that we don't support `uint`. But do we support `uint32`?

Reply:
Updated to use int64